.scripts/build_steps.sh

```diff
@@ -19,7 +19,7 @@ conda-build:
 
 CONDARC
 
-conda install --yes --quiet conda-forge-ci-setup=2 conda-build -c conda-forge
+conda install --yes --quiet conda-forge-ci-setup=2 "conda-build<3.18" -c conda-forge
```

Comment:
Sorry about that, it seems it's bugs all the way down! This version worked on `master`:
```suggestion
conda install --yes --quiet conda-forge-ci-setup=2 "conda-build=3.18.11" -c conda-forge
```
Taken from https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=121043&view=logs&j=1bf226d3-0e2f-52d8-fa93-7d9e633347b3&t=ec8d466d-e3b4-5115-4f06-222398f91e6c